docusaurus/docs/identities/enrolling.md

```diff
@@ -57,15 +57,15 @@ confident you understand the risks involved in doing so.
 
 ### 3rd Party CA - One Time Token
 
-This process is similar to the One Time Token flow from above. This flow expects that a private key and certificate have
+The flow allows you to pre-create identities for a 3rd party CA with distinct role attributes. This flow is similar to the One Time Token flow from above except that it expects that a private key and certificate have
 already been created on or distributed to the machine that is about to enroll and that the certificate presented is
-signed by a [third party CA](../manage/pki#third-party-ca-optional) already validated in the Ziti Controller.
+issued by a federated [third party CA](../manage/pki#third-party-ca-optional).

```

Comment:
remove federated. it's not a term i've seen used elsewhere and it's not the process that one needs to follow. one creates a CA then one verifies that CA. those are the terms we should stick with. validated implies created, the original phrasing was better